src/pages/ReportDetailsPage.tsx

```diff
@@ -1068,8 +1068,18 @@ function ReportDetailsPage({policies, report, route, reportMetadata}: ReportDeta
                         <MenuItem
                             key={CONST.REPORT_DETAILS_MENU_ITEM.DELETE}
                             icon={Expensicons.Trashcan}
-                            title={caseID === CASES.DEFAULT ? translate('common.delete') : translate('reportActionContextMenu.deleteAction', {action: requestParentReportAction})}
-                            onPress={() => setIsDeleteModalVisible(true)}
+                            title={
```

Comment:
@allgandalf the logic to display the `Delete report` action should be added to the new secondary report action [here](https://github.com/Expensify/App/blob/ae74f299bc957064f4b31b29a3eb70fae4dc1c9a/src/libs/ReportSecondaryActionUtils.ts). The condition to show it is outlined in the design doc [here](https://docs.google.com/document/d/1JZLr_5gX348cMZXNr6HUwi4QscOcDg3IY368FMAu3H4/edit?tab=t.0#bookmark=id.f2dbxoo8l2jl)

and then the UI should be built [here](https://github.com/Expensify/App/blob/ae74f299bc957064f4b31b29a3eb70fae4dc1c9a/src/components/MoneyReportHeader.tsx#L490-L633)